torch/csrc/jit/python/init.cpp

```diff
@@ -215,6 +216,20 @@ void initJITBindings(PyObject* module) {
             return PeepholeOptimizeONNX(graph, opset_version, fixed_batch_size);
           })
       .def("_jit_pass_onnx_preprocess", PreprocessForONNX)
+      .def(
+          "_jit_pass_onnx_deduplicate_initializers",
+          [](std::shared_ptr<Graph>& graph,
+             std::map<std::string, IValue>& paramsDict,
+             bool is_train,
+             int opset_version) {
+            DeduplicateInitializers(
+                graph,
+                paramsDict,
+                is_train,
+                opset_version); // overload resolution
+            return paramsDict;
```

Comment:
A bit confusing to both modify the input and return something: seeing a return value suggests this is a pure function.
I suggest we remove the return value so it's clearer that the input is modified.
Or don't modify the input and make a copy if you prefer.

Reply:
Updated, I thought this was pybind restriction, from comments of other places in the file. But it seems just passing reference works.